src/events/wheel.ts

```diff
@@ -22,10 +22,26 @@ export function wheelHandler(scrollbar: I.Scrollbar) {
   });
 }
 
-// Normalizing wheel delta
+const getStandardDeltaScale = () => {
+  const UA = navigator.userAgent;
+  const isMac = navigator.platform.toUpperCase().indexOf('MAC') >= 0;
+  const isEdge = UA.indexOf('Edge/') > -1;
+  const isFF = UA.indexOf('Firefox') > -1;
+  const isChrome = navigator.userAgent.toLowerCase().indexOf('chrome') > -1;
+
+  if (isMac) {
+    if (isFF) return 0.85;
+  } else {
+    if (isFF) return 0.35;
+    if (isEdge) return 0.91;
+    if (isChrome) return 0.44;
```

Comment:
Why are these magic numbers?

Reply:
DELTA_SCALE.STANDARD = 1 is too generic, need to be normalized.
After trial and error, i have identified that these 'magic numbers' normalize the wheeldelta.
If you have a windows and osx computer, try removing the function, test on chrome and you will notice a huge difference.

Reply:
It's meaningless to compare cross platform, since the input devices and drivers are different. I think we may need to compare `wheelEvent.deltaY` with actual scrolling distance `Δwindow.scrollTop` to get the precise ratio.

Reply:
I understand the simplicity of my approach, but your library needs a normalization function.
The scrolling behavior is totally different from osx and windows.
I have however tested on a lot of devices and browsers, and the wheeldelta is more accurate.

Reply:
Hey dude, look, we are not physicists. We can't decide the value only through observations and experiments. We need something more logic to prove that the values are correct and reasonable.

AFAIK, scrolling distance is related to `wheelDelta` on whatever the platform is. Hence we can use `ratio = ΔscrollTop / wheelEvent.deltaY` to get the scale ratio.